apis/telemetry/v1beta1/logpipeline_validation_test.go

```diff
@@ -123,8 +100,7 @@ func TestBothValueAndValueFromPresent(t *testing.T) {
 				},
 			},
 		}}
-	vc := getLogPipelineValidationConfig()
-	err := logPipeline.validateOutput(vc.DeniedOutPutPlugins)
+	err := logPipeline.validateOutput()
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "http output host must have either a value or secret key reference")
```

Comment:
nit (not part to be implemented as part of this PR):
lets use errors.Is to match the error and not string matching

Reply:
Agree, will do it in a separate PR.